_docs/floodgate/linking.md

```diff
@@ -8,6 +8,9 @@ permalink: /floodgate/linking/
 Instructions and information about how to link can also be found here: https://link.geysermc.org/
 
 Before we introduced Global Linking, you always had to link your Java and Bedrock account on every individual server you visited (that has Floodgate). Global Linking is here to fix that problem. Link once, join everywhere.<br>
+
+Keep in mind: While having your accounts linked, you will use the Java accounts' location, inventory data, and achievements etc. regardless of which platform you signs in from (therefore, "synchronising" the player data). The player data from the Bedrock account will not be accessible until you unlink again - therefore, you should transfer everything (enderchest contents, items, armor) to the Java account before linking to not "lose" your Bedrock progress. If you forgot to do this, you can unlink, transfer everything over, and link again. 
```

Comment:
Java account's

Since it's a singular player we're talking about.

Also enderchest -> ender chest

Reply:
fixed, thanks for the correction